### PR TITLE
Mcp 2772 exam order slack 24hr param

### PR DESCRIPTION
**Pull request by ianwolson**
## What was the problem?
We needed a way to slack all of the exam orders not processed by MAS that are older than 24 hours.

Associated tickets or Slack threads:

- [MCP-2772](https://amida.atlassian.net/browse/MCP-2772)

## How does this fix it?[^1]
Created a JPA repository method to get all exam orders with orderedAt = null and createdAt older than 24 hours.

## How to test this PR
- Step 1
- Step 2[^secrel]

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

File: svc-lighthouse-api/src/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM eclipse-temurin:17-jre-alpine
```
Comment (jbregstone) [inline svc-lighthouse-api/src/docker/Dockerfile:1]: These changes don't seem related to the ticket

File: domain-rrd/rrd-api-controller/src/main/java/gov/va/vro/controller/rrd/ClaimMetricsController.java [hunk 2]
```diff
@@ -86,7 +86,13 @@ public ResponseEntity<List<ExamOrderInfoResponse>> examOrderSlack(
       Integer page, Integer size, Boolean notOrdered) throws ClaimProcessingException {
     ExamOrderInfoQueryParams params =
         ExamOrderInfoQueryParams.builder().page(page).size(size).notOrdered(notOrdered).build();
-    ExamOrdersInfo examOrdersInfo = claimMetricsService.findExamOrderInfo(params);
+    ExamOrdersInfo examOrdersInfo = claimMetricsService.findExamOrderInfoOlderThan24(params);
+    if (examOrdersInfo.getExamOrderInfoList().size() < 1) {
+      throw new ClaimProcessingException(
```
Comment (jbregstone) [inline domain-rrd/rrd-api-controller/src/main/java/gov/va/vro/controller/rrd/ClaimMetricsController.java:91]: Seems like an odd place to throw a `ClaimProcessingException`. I would probably just return an empty list instead.

**Review by jbregstone — APPROVED** [review]
Please add unit test if it' snot to difficult to mock the local time component.

**Comment by ianwolson** [comment]
> Please add unit test if it's not to difficult to mock the local time component.

Just added a test, let me know what you think.

**Comment by jbregstone** [comment]
> > Please add unit test if it's not to difficult to mock the local time component.
> 
> @jbregstone I added a test, let me know what you think.

Looks good!

**Review by jbregstone — APPROVED** [review]
